shell/browser/api/electron_api_web_contents.cc

```diff
@@ -2539,6 +2539,60 @@ void WebContents::SetWebRTCIPHandlingPolicy(
   web_contents()->SyncRendererPrefs();
 }
 
+v8::Local<v8::Value> WebContents::GetWebRTCUDPPortRange(
+    v8::Isolate* isolate) const {
+  uint16_t min = 0, max = 0;
+  auto* prefs = web_contents()->GetMutableRendererPrefs();
+
+  min = prefs->webrtc_udp_min_port;
+  max = prefs->webrtc_udp_max_port;
+
+  gin_helper::Dictionary dict = gin::Dictionary::CreateEmpty(isolate);
+  dict.Set("min", static_cast<uint32_t>(min));
+  dict.Set("max", static_cast<uint32_t>(max));
+  return dict.GetHandle();
+}
```

Comment:
We don't need to create a separate variable for this :)

```suggestion
v8::Local<v8::Value> WebContents::GetWebRTCUDPPortRange(
    v8::Isolate* isolate) const {
  auto* prefs = web_contents()->GetMutableRendererPrefs();

  gin_helper::Dictionary dict = gin::Dictionary::CreateEmpty(isolate);
  dict.Set("min", static_cast<uint32_t>(prefs->webrtc_udp_min_port));
  dict.Set("max", static_cast<uint32_t>(prefs->webrtc_udp_max_port));
  return dict.GetHandle();
}
```